### PR TITLE
Pnnl Requests, Part 2

### PR DESCRIPTION
**Pull request by jasondegraw**
Both @fieldkm and I have made changes here, so this one is on you, @nllong. I'd like to do the change to spaces rather than tabs after this PR gets merged, so let's hold of on changes for a bit. Don't delete the branch after the merge.

### CODE REVIEW

File: BuildingSync.xsd [hunk 1]
```diff
@@ -14081,6 +14088,38 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
...
+						<xs:sequence>
+							<xs:element name="TypicalOccupantUsageValue" type="xs:decimal"/>
+							<xs:element name="TypicalOccupantUsageUnits">
+								<xs:simpleType>
+									<xs:restriction base="xs:string">
+										<xs:enumeration value="Hours per day"/>
+										<xs:enumeration value="Hours per week"/>
+										<xs:enumeration value="Hours per month"/>
+										<xs:enumeration value="Hours per year"/>
+										<xs:enumeration value="Days per week"/>
```
Comment (nllong) [inline BuildingSync.xsd]: Are days per * really useful? Are this 24 hours/per day, x days per week? or is it 8 hours/day?

I think that we want hours to be the common unit so perhaps remove days per *, weeks per *, and months per year.

Comment (nllong) [inline BuildingSync.xsd]: i talked with degraw and this has been resolved. the user is able to have more than one unit.

**Comment by fieldkm** [comment]
Jason to merge this now?

**Comment by jasondegraw** [comment]
Yes, I will.

**Pull request merged**